homeassistant/components/zwave_js/switch.py

```diff
@@ -46,6 +46,9 @@ class ZWaveSwitch(ZWaveBaseEntity, SwitchEntity):
     @property
     def is_on(self) -> bool:
         """Return a boolean for the state of the switch."""
+        if self.info.primary_value.value is None:
+            # guard missing value
+            return False
```

Comment:
Dito

Reply:
And same reply here, I'm not sure if we can return None to HA here or that it really needs a boolean value.
If it accepts None (undefined state) the user can toggle the switch to make the value come "alive"